profiles/package.mask

```diff
@@ -33,6 +33,18 @@
 
 #--- END OF EXAMPLES ---
 
+# Jaco Kroon <jaco@uls.co.za> (2023-09-19)
+# I intend to abandon support for DAHDI.  Maintenance is turning into an
+# absolute nightmare.  If you still use this, please let me know so that we can
+# make a plan, I'm happy to assist where I can, but I can't justify the sheer
+# amount of effort that's going into this given that I no longer use this.
+# Various open bugs, upstream isn't coming to the party.
+# Intended removal on 2024-03-01.
```

Comment:
Reference the dedicated bug here too. Giving motivation and concrete reason is great in this mask as this will be displayed for all users with these listed packages installed, but the 1st person POV may come out as a bit intimidating. Try to write it more passively, but directing the users towards the dedicated bug to raise their voice and help out.

Reply:
I tried.  But I'm afraid it's not at all easy to completely get rid of my frustration here.  Should be marginally better now.